ocs_ci/helpers/cephfs_stress_helpers.py

```
@@ -286,6 +316,24 @@ def _wait_with_retry(self, resource, state, timeout):
         """
         wait_for_resource_state(resource, state=state, timeout=timeout)
 
+    def _read_env_var_from_spec(self, env_list, name, default=""):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `_read_env_var_from_spec`

(ANN202)